src/test/java/com/automationexercise/tests/Tests/TestCasesPageTest.java

```diff
@@ -0,0 +1,37 @@
+package com.automationexercise.tests.Tests;
+
+import com.automationexercise.DriverManager.BrowserFactory;
+import com.automationexercise.DriverManager.DriverManager;
+import com.automationexercise.Pages.HomePage;
+import com.automationexercise.Pages.TestCasesPage;
+import com.automationexercise.Utilities.Utility;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import static com.automationexercise.DriverManager.DriverFactory.createInstance;
+import static com.automationexercise.DriverManager.DriverManager.*;
+import static com.automationexercise.Utilities.Utility.openWebsite;
+
+public class TestCasesPageTest  {
+    WebDriver driver = BrowserFactory.CHROME.createDriver();
+    @BeforeMethod(alwaysRun = true)
+    public void setUp() {
+        /*createInstance("chrome");
+        openWebsite("https://automationexercise.com/");*/
+        driver.get("https://automationexercise.com/");
```

Comment:
You need not to call **driver.get("https://automationexercise.com/");**
You just initialize the driver using **createInstance("chrome");** then you can open the url using **openWebsite("https://automationexercise.com/");**